cmd/snap/cmd_pack.go

```diff
@@ -45,7 +47,9 @@ func init() {
 		longPackHelp,
 		func() flags.Commander {
 			return &packCmd{}
-		}, nil, nil)
+		}, map[string]string{
+			"check-skeleton": i18n.G("Validate snap directory only"),
```

Comment:
i feel that description does a poor job of explaining the flag.
At the same time, explaining the flag is probably more text than should be in a flag's  help.
So: please explain what the flag does in detail, in the command's help; then a short flag description that relies on that explanation will be appropriate (but even then I don't think this is the right text).